app/templates/components/footer-main.hbs

```diff
@@ -52,7 +52,7 @@
   </div>
   <div class="ui inverted section divider"></div>
   <div class="ui stackable grid">
-    <div class="fourteen wide column left aligned">
+    <div class="thirteen wide computer twelve wide tablet column left aligned">
```

Comment:
Why was original _fourteen - two_ width configuration changed even for large screens:
There was a slight overflow even on computer sized screens for language with a _longer_ name.
![image](https://user-images.githubusercontent.com/17252805/27449218-322ba468-57a5-11e7-93aa-58226a209f87.png)

Not significant enough to cause a scroll bar, but was pushing it to the limit.